tools/docker/Dockerfile.centos8.1

```diff
@@ -83,7 +90,7 @@ COPY toolchains/rust /tmp/rust
 WORKDIR /tmp
 RUN cd rust && ./build.sh && rm -rf /tmp/rust
 ENV PATH="/opt/occlum/toolchains/rust/bin:$PATH"
-ENV LD_LIBRARY_PATH="$LD_LIBRARY_PATH:/opt/intel/sgxsdk/sdk_libs"
```

Comment:
Why replacing the path instead of inheriting?

Reply:
This makes the `LD_LIBRARY_PATH =:/opt/intel/sgxsdk/sdk_libs`, the first element is `(null)`.
If I installed <glibc-for-occlum-dir> in Docker container, then I `cd` into the dir, any bash commands will fail because the commands will use the ld.so or libs in the <glibc-for-occlum-dir>, which is the current directory.